src/selectors/theme.ts

```diff
@@ -23,6 +23,7 @@ import publishMode from '../util/publishMode'
 
 /** Gets the theme. */
 const theme = (state: State) => {
+  // return 'Dark'
```

Comment:
Is this comment intentional? I assume it was added for debugging and not meant to be included.

Reply:
You are right, thanks for catching this!